bin/etc-update

```diff
@@ -631,6 +631,10 @@ do_merge() {
 					else
 						chown --reference="${ofile}" "${mfile}"
 						chmod --reference="${ofile}" "${mfile}"
+						${selinux} && {
+							chcon --reference="${ofile}" "${mfile}" || \
+								warn "Failed setting SELinux security label on ${mfile}"
```

Comment:
This feels redundant. Doesn't chcon output an error message on failure? We don't add a special warning when chmod fails above.

Reply:
Good point. I'll remove this extraneous warning.